package.json

```diff
@@ -2,7 +2,7 @@
   "name": "tudelft-spacefinder",
   "private": true,
   "scripts": {
-    "data": "node scripts/data/index.js",
+    "data": "tsx --tsconfig ./tsconfig.data-scripts.json scripts/data/index.ts",
```

Comment:
This command runs for me without the `tsconfig` flag.
Usually I use [tsm](https://github.com/lukeed/tsm) however [jiti](https://github.com/unjs/jiti) seems to be an even better choice here, Nuxt team is behind jiti, it's small and has no dependencies.